spacify-engine/src/main/java/org/uci/spacifyEngine/controllers/SpacifyEngine.java

```diff
@@ -35,6 +36,14 @@ public List<ReservationEntity> getAllReservations() {
         return this.reservationService.getAllReservatons();
     }
 
+    @Autowired
+    private AvailableSlotsService availableSlotsService;
```

Comment:
This also needs to be in specify-portal not specify engine. You can create a new ReservationController there